server/controllers/allocationsController.ts

```diff
@@ -178,8 +178,23 @@ export default class AllocationsController {
       offenderManagerCode,
       convictionId
     )
-    res.render('pages/officer-view', {
-      title: 'Officer view',
+    res.render('pages/overview', {
+      title: 'Overview',
+      data: response,
+      crn: response.crn,
+      convictionId: response.convictionId,
+    })
+  }
+
+  async getActiveCases(req: Request, res: Response, crn, offenderManagerCode, convictionId) {
+    const response: OffenderManagerPotentialWorkload = await this.allocationsService.getCaseAllocationImpact(
```

Comment:
I know this endpoint will change as we'll be having a new active cases endpoint in hmpps-allocations but just using this for now as it brings back the data we need for the first part of the active cases page.